HomeWork2

```diff
@@ -0,0 +1,18 @@
+class Main {
+    public static void main(String[] args) {
+        aisuluu("Even");
+        aisuluu("Odd");
+
+    }
+
+    public static void aisuluu(String word) {
```

Comment:
name of your method should show what this method do exactly! It should start with verb or be a verb! It is a good practice because you will be able to understand the purpose of the method without reading a code into method